src/data/stablecoinList.ts

```diff
@@ -0,0 +1,63 @@
+export default [
+  {
+    // USD coins
+    priority: 1,
+    addresses: {
+      1: [
```

Comment:
Do not hardcode the network. We have an enum, we can say MAINNET here

Reply:
this is coming directly from the JS file i was given, i think 1 is better to grab with our networkId